storage/cloudspanner/tree_storage.go

```diff
@@ -209,7 +210,7 @@ func (t *treeTX) currentSTH(ctx context.Context) (*spannerpb.TreeHead, error) {
 
 func (t *treeTX) writeRev(ctx context.Context) (int64, error) {
 	if err := t.getLatestRoot(ctx); err == storage.ErrTreeNeedsInit {
-		return 0, nil
+		return t._writeRev, nil
```

Comment:
Is it ok to return -1 rather than 0 here now?

I think the sequencer / admin stuff is where the most tricksy `ErrTreeNeedsInit` support is located, BTW.

Reply:
Yes, but rather than having this be another edgecase, I prefer this style.
This is also the way that the mysql impl does it.